### PR TITLE
[TR] PIM-4816: add a formater to format bytes

### PR DESCRIPTION
**Pull request by solivier**
| Q                  | A
| ------------------ | ---
| Migration script?  | y
| Behats             | running
| Specs & Static     | y
| Changelog updated  | n

### CODE REVIEW

File: src/Akeneo/Component/FileStorage/Formater/BytesFormater.php [hunk 1]
```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Akeneo\Component\FileStorage\Formater;
+
+/**
+ * Convert bytes into human readable value
+ *
+ * @author Olivier Soulet <olivier.soulet@akeneo.com>
```
Comment (skeleton) [inline src/Akeneo/Component/FileStorage/Formater/BytesFormater.php]: missing @copyright & @license

File: src/Akeneo/Bundle/FileStorageBundle/Resources/config/formaters.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+parameters:
+    akeneo_file_storage.file_storage.formater.bytes.class:  Akeneo\Component\FileStorage\Formater\BytesFormater
```
Comment (skeleton) [inline src/Akeneo/Bundle/FileStorageBundle/Resources/config/formaters.yml]: remove extra space after ":"

**Comment by skeleton** [comment]
no behat ?
GTM after very tiny comment.

**Comment by solivier** [comment]
no behat it's currently only used in EE, behat exists in EE

**Comment by grena** [comment]
GTM

**Pull request merged**